src/serlo-editor/plugins/text/hooks/use-suggestions.tsx

```diff
@@ -140,6 +141,13 @@ export const useSuggestions = (args: useSuggestionsArgs) => {
     SELECT_UP: handleSelectionChange('up'),
     SELECT_DOWN: handleSelectionChange('down'),
     INSERT: handleSuggestionInsert,
+    ESCAPE: () => {
+      // We delete the line with the slash and therefore, close the suggestions
+      // upon next render
+      if (closure.current.showSuggestions) {
+        editor.deleteBackward('line')
+      }
+    },
```

Comment:
To stay in line with the current code style, I'd prefer to declare this as a function, and then the comment would also not be necessary.

```suggestion
    ESCAPE: handleCloseSuggestions,
```